### PR TITLE
Permissions UI refactor

### PR DESCRIPTION
**Pull request by cnweaver**
This set of changes fixes the permissions management pages to be clearer and more robust. The primary change is to provide separate add/remove buttons for each permission so that it is always clear what pressing any given button will do. This and related bug fixes remove inconsistency between the front end and back end. Help text has been correspondingly updated, and some page titles fixed. 

Related, although not directly connected to the above changes, various navigation has been improved, including links back from individual topic pages to the owning groups' topic summary pages, and from each Hopskotch page to the top-level Hopskotch page.

### CODE REVIEW

File: scimma_admin/hopskotch_auth/templates/hopskotch_auth/hopskotch.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{% extends "hopskotch_auth/base.html" %}
+
+{% block nav-section %}
+	<span class="navbar-brand">—</span>
+	<a class="navbar-brand" href="/hopauth">Hopskotch</a>
+{% endblock %}
```
Comment (jnation3406) [inline scimma_admin/hopskotch_auth/templates/hopskotch_auth/hopskotch.html:6]: newline

File: scimma_admin/hopskotch_auth/templates/hopskotch_auth/manage_topic.html [hunk 2]
```diff
@@ -66,108 +67,55 @@ <h2>Manage Topic</h2>
...
-        <h2>Available Groups</h2>
+        <h2>Available Permissions</h2>
         <div id="cur_permissions">
             <table class="table table-hover" id="avail_table">
                 <thead>
                     <tr>
-                        <th scope="col">Group Name</th>
-                        <th scope="col">Add</th>
+                        <th scope="col">Group</th>
+                        <th scope="col">Permission</th>
```
Comment (jnation3406) [inline scimma_admin/hopskotch_auth/templates/hopskotch_auth/manage_topic.html]: You call them Permissions here, but in the `manage_credential.html` page you called the Operations?

File: scimma_admin/hopskotch_auth/views.py [hunk 3]
```diff
@@ -407,33 +406,32 @@ def manage_topic(request, topicname) -> HttpResponse:
...
-            perms_result = engine.get_group_permissions_for_topic(request.user, group.name, topicname)
-            if not perms_result:
-                return redirect_with_error(request, 'manage_topic', perms_result.err(), 'index')
-            ops = [op.operation for op in perms_result.ok()]
-            cleaned_added.append({'name': group.name,
-                                  'read': KafkaOperation.Read in ops,
-                                  'write': KafkaOperation.Write in ops})
-        else:
-            cleaned_available.append(group.name)
+        for op in [KafkaOperation.Read, KafkaOperation.Write]:
```
Comment (jnation3406) [inline scimma_admin/hopskotch_auth/views.py:427]: here's another place your using operation/permission interchangeably

**Review by jnation3406 — APPROVED** [review]
Looks good to me. I think handling it per permission vs just read/write is a nice improvement.

File: scimma_admin/hopskotch_auth/templates/hopskotch_auth/manage_topic.html (hunk 2 above)
Comment (cnweaver) [reply scimma_admin/hopskotch_auth/templates/hopskotch_auth/manage_topic.html]: Thanks for catching that; fixed in 03c6e9c.

File: scimma_admin/hopskotch_auth/views.py (hunk 3 above)
Comment (cnweaver) [reply scimma_admin/hopskotch_auth/views.py:427]: It's really too late to change the name of the KafkaOperation type unless it's truly important, and while consistency is nice inside the code, I think it's less important that keeping the database schema stable.

**Pull request merged**